test/integration/consul-container/test/wanfed/wanfed_peering_test.go

```diff
@@ -1,7 +1,7 @@
 // Copyright (c) HashiCorp, Inc.
 // SPDX-License-Identifier: MPL-2.0
 
-package peering
+package wanfed
```

Comment:
I noticed another package is called `peering` so I renamed this to match the directory name.